src/test/java/com/google/api/codegen/testsrc/library_gapic.yaml

```diff
@@ -227,6 +227,28 @@ interfaces:
     resource_name_treatment: STATIC_TYPES
     sample_code_init_fields:
     - series_uuid.series_string=foobar
+    code_sample_values:
+      - id: review
```

Comment:
The convention in our yamls is for the `-` lines to start at the same indentation as the key.

Reply:
Fixed